grafana-plugin/src/components/LabelsTooltipBadge/LabelsTooltipBadge.tsx

```diff
@@ -36,3 +38,32 @@ export const LabelsTooltipBadge: FC<LabelsTooltipBadgeProps> = ({ labels, onClic
       }
     />
   ) : null;
+
+interface LabelBadgesProps {
+  labels: Array<components['schemas']['LabelPair']>;
+  maxCount?: number;
+}
+
+export const LabelBadges: React.FC<LabelBadgesProps> = ({ labels = [], maxCount = 3 }) => {
+  const renderer = (values: Array<components['schemas']['LabelPair']>) => {
+    return (
+      <HorizontalGroup>
+        {values.map((label) => (
+          <LabelTag key={label.key.id} label={label.key.name} value={label.value.name} />
+        ))}
+      </HorizontalGroup>
+    );
+  };
+
+  return (
+    <HorizontalGroup spacing="sm">
+      {renderer(labels.slice(0, maxCount))}
```

Comment:
Why do we need renderer function instead of doing this directly?
```
<HorizontalGroup>
  {labels.slice(0, maxCount).map((label) => (
          <LabelTag key={label.key.id} label={label.key.name} value={label.value.name} />
        ))}
</HorizontalGroup>
```

Reply:
Because we reuse it

Reply:
![image](https://github.com/user-attachments/assets/efbab017-9cb6-4001-8963-eb823a8d21ef)

Reply:
Ah, missed it! all good